lib/tty/command/printers/pretty.rb

```diff
@@ -38,14 +38,20 @@ def print_command_exit(cmd, status, runtime, *args)
         # Write message out to output
         #
         # @api private
-        def write(message, uuid = nil)
+        def write(message, uuid = nil, dry_run = false)
```

Comment:
I guess it's not a problem to have this here, as it defaults to `false` not to break the `Pretty` printer. Also, I decided to modify this because the new one is based on this so I wanted to avoid duplicating code.